src/org/rascalmpl/interpreter/Evaluator.java

```diff
@@ -1437,13 +1437,13 @@ public GlobalEnvironment getHeap() {
     }
 
     @Override	
-    public boolean runTests(IRascalMonitor monitor) {
+    public boolean runTests(IRascalMonitor monitor, String optionalModuleName) {
```

Comment:
can we not make a overload for this? to avoid having a `null`  parameter that users now have to pass in?